frontend/src/core/utils/fluent/isPluralElement.js

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+
+import { CLDR_PLURALS } from 'core/plural';
+
+import type { PatternElement } from './types';
+
+
+/**
+ * Return true when AST element represents a pluralized string.
+ *
+ * Keys of all variants of such elements are either CLDR plurals or numbers.
```

Comment:
In https://bugzilla.mozilla.org/show_bug.cgi?id=1385863, we decided to drop `other` from the tested plural categories. Notably, 'cause there's often `[other]` used in a non-plural-form sense.